README.md

```diff
@@ -32,7 +32,7 @@ This lightweight SDK connects Android to Zcash. It welds together Rust and Kotli
 
 ## Requirements
 
-This SDK is designed to work using a light client server [lightwalletd](https://github.com/zcash-hackworks/lightwalletd)
+This SDK is designed to work using a light client-server [lightwalletd](https://github.com/zcash-hackworks/lightwalletd)
```

Comment:
This is a server for "light clients" so a hyphen here wouldn't be needed but maybe we should reword it so it's more clear? It is probably fine to shorten this to:
 
>This SDK is designed to work with [lightwalletd](https://github.com/zcash/lightwalletd)